MvvmCross-Forms/MvvmCross.Forms.Droid/Views/MvxFormsAppCompatActivity.cs

```diff
@@ -26,7 +26,7 @@ protected MvxFormsAppCompatActivity()
         {
             BindingContext = new MvxAndroidBindingContext(this, this);
             this.AddEventListeners();
-            _resourceAssembly = Assembly.GetCallingAssembly();
+            _resourceAssembly = this.GetType().Assembly;
```

Comment:
I don't think this is going to work. We need to get the actual assembly of the app project. @johnnywebb thoughts?

Reply:
@martijn00 you are correct. The objective here is to obtain the project assembly containing the android resources. In most cases, the app assembly contains a MainActivity derived from MvxFormsAppCompatActivity; hence, why `Assembly.GetCallingAssembly` is use in the constructor. [More info can be found here](https://github.com/MvvmCross/MvvmCross/blob/8146d2ca600e043d3e1dfbe0ef096dbf26332973/docs/_documentation/platform/forms/xamarin-forms-customization.md)

`this.GetType().Assembly` will return the `MvvmCross.Forms.Droid` assembly and the droid resources, such as images, will fail to load.

Reply:
@johnnywebbbankmobile I don't think that's true.... this.GetType().Assembly will get the Type for the sub-type (ie the class that inherits from MvxFormsAppCompatActivity, not MvFormsAppCompatActivity), so will pick up the correct assembly

Assembly.GetCallingAssembly will pick up the assembly of the calling method. This fails because if you use MvxFormsAppCompatActivity<TViewModel> because it picks up the ctor for that class which is in MvvmCross.Forms.Droid.

I'm not fussed about this particular change as it didn't turn out to be the cause of the Droid XF not rendering but I don't believe the current code is correct and should be fixed

Reply:
ah yeah you are right @nickrandolph . 👍 good catch, I'll fix it.

Reply:
@nickrandolph good catch. I fixed here: https://github.com/MvvmCross/MvvmCross/pull/2461